Sources/MapboxCoreNavigation/RouterDelegate.swift

```diff
@@ -63,6 +66,8 @@ public protocol RouterDelegate: AnyObject, UnimplementedLogging {
      */
     func router(_ router: Router, didFailToRerouteWith error: Error)
 
+    // MARK: Progressing the Route and Arriving Events
```

Comment:
Did you mean “processing” rather than “progressing”?

Reply:
I meant "progressing" as following methods are called at various stages during user progress on the route. Also wanted to align `RouterDelegate` and `NavigationServiceDelegate` sections, so I'll use `Monitoring Route Progress and Updates` here too.